packages/embark/src/cmd/cmd.js

```diff
@@ -170,42 +167,6 @@ class Cmd {
       });
   }
 
-  run2() {
```

Comment:
Don't we still use that?

Reply:
NVM, just saw that you changed run2 ro run and removed run2. 
If possible, make it clearer to everyone that you changed that, since we rely on `run2` right now.